<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>738</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>계정별검색보기</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>3</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>특별회계 은행 계정별 검색보기</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색대상년도</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="result_tableWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>140</y>
     <width>891</width>
     <height>571</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="serch_Button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_Button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>20</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>100</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>은 행 명</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bank_combo_widget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>97</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="year_combo_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>76</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_cost_Button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택행 수정 저장하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="serch_print_Button">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>프린트</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bank_combo_widget</tabstop>
  <tabstop>serch_Button</tabstop>
  <tabstop>result_tableWidget</tabstop>
  <tabstop>reset_Button</tabstop>
  <tabstop>close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>reset_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>884</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_cost_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>edit_cost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serch_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>special_bank_account_view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serch_print_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>serch_special_account_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>special_bank_account_view()</slot>
  <slot>reset()</slot>
  <slot>edit_cost()</slot>
  <slot>serch_special_account_print()</slot>
 </slots>
</ui>
